Packs/Jira/Integrations/JiraV3/JiraV3.py

```diff
@@ -630,6 +632,23 @@ def delete_issue(self, issue_id_or_key: str) -> requests.Response:
             resp_type='response',
         )
 
+    def update_assignee(self, issue_id_or_key: str, assignee_body: Dict[str, Any]) -> requests.Response:
+        """This method is in charge of assigning the issue to the specific user.
+
+        Args:
+            issue_id_or_key (str): The id or the key of the issue to delete.
+            assignee_body (Dict[str, Any]): Dictionary containing assignee_id / assignee
+
+        Returns:
+            requests.Response: The raw response of the endpoint.
+        """
+        return self.http_request(
+            method='PUT',
+            url_suffix=f'rest/api/{self.api_version}/issue/{issue_id_or_key}/assignee',
+            json_data=assignee_body,
+            resp_type="text"
```

Comment:
According to the [Cloud Documentation](https://developer.atlassian.com/cloud/jira/platform/rest/v3/api-group-issues/#api-rest-api-3-issue-issueidorkey-assignee-put), the response body is empty, therefore, I believe this should be **resp_type='response'**

Reply:
Fixed